ntoskrnl/cc/view.c

```diff
@@ -158,7 +158,7 @@ CcRosTraceCacheMap (
 NTSTATUS
 CcRosFlushVacb (
     _In_ PROS_VACB Vacb,
-    _In_ PIO_STATUS_BLOCK Iosb)
+    _Out_opt_ PIO_STATUS_BLOCK Iosb)
```

Comment:
Show me where in this function `Iosb` parameter gets written into.

Reply:
Really?
https://github.com/reactos/reactos/blob/c3dbcde1e45ab4c5c2d977a92fda1160bb7571e6/ntoskrnl/cc/view.c#L178-L181

Reply:
For some reason I don't think it's a write operation.

Reply:
What do you think it is, then?

Reply:
`MmFlushSegment` writes data into the I/O status block the information field and status code respectively, only if such block is actually filled to the call. 
https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/mm/section.c;hb=d07203ecb700a11c21608448d2326ea6a6b3f368#l4662

In the case of `CcRosFlushVacb` we write back to the I/O status block whatever stuff and the caller can do whatever it wants with this variable after it's been written to it. With that said it'd certainly make sense to treat this argument as `_Out_`.

Reply:
That was not clear for me, thanks for the explanation. :+1: